WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginEmailFragment.java

```diff
@@ -323,4 +369,54 @@ protected void onLoginFinished() {
         AnalyticsUtils.trackAnalyticsSignIn(mAccountStore, mSiteStore, true);
         mLoginListener.loggedInViaSocialAccount(mOldSitesIDs);
     }
+
+    @Override
+    public void onConnected(Bundle bundle) {
+        AppLog.d(T.NUX, LOG_TAG + ": Google API client connected");
+    }
+
+    @Override
+    public void onConnectionFailed(@NonNull ConnectionResult connectionResult) {
+        AppLog.d(T.NUX, LOG_TAG + ": Google API connection result: " + connectionResult);
+    }
+
+    @Override
+    public void onConnectionSuspended(int i) {
+        AppLog.d(T.NUX, LOG_TAG + ": Google API client connection suspended");
+    }
+
+    public void getEmailHints() {
+        HintRequest hintRequest = new HintRequest.Builder()
+                .setHintPickerConfig(new CredentialPickerConfig.Builder()
+                        .setShowCancelButton(true)
+                        .build())
+                .setEmailAddressIdentifierSupported(true)
+                .build();
+
+        PendingIntent intent = Auth.CredentialsApi.getHintPickerIntent(mGoogleApiClient, hintRequest);
+
+        try {
+            startIntentSenderForResult(intent.getIntentSender(), EMAIL_CREDENTIALS, null, 0, 0, 0, null);
+        } catch (IntentSender.SendIntentException exception) {
+            AppLog.d(T.NUX, LOG_TAG + "Could not start email hint picker" + exception);
+        }
+    }
+
+    @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent data) {
+        super.onActivityResult(requestCode, resultCode, data);
+
+        if (requestCode == EMAIL_CREDENTIALS) {
+            if (resultCode == RESULT_OK) {
+                Credential credential = data.getParcelableExtra(Credential.EXTRA_KEY);
+                mEmailInput.getEditText().setText(credential.getId());
+                next(getCleanedEmail());
+            } else {
+                hasDismissedEmailHints = true;
+                EditTextUtils.showSoftInput(mEmailInput.getEditText());
```

Comment:
The issue with the keyboard not showing up still happens on my end.

It seems to be a timing thing so, I managed to work around it by delay-posting the call like this:

```
mEmailInput.getEditText().postDelayed(new Runnable() {
    @Override
    public void run() {
        if (isAdded()) {
            EditTextUtils.showSoftInput(mEmailInput.getEditText());
        }
    }
}, 400);
```

Can we perhaps add something like that? WDYT?

Reply:
I'm not a fan of adding arbitrary delays like this, but I'm unable to reproduce the behavior so I'm not sure what is the root cause.  I thought it might be related to the email hint picker intent, but I still can't reproduce the behavior when upping the animation scale.

I added the delay in a34d6a92ba44539b5d5521f58c31857c24d687d0.

Reply:
> I'm not a fan of adding arbitrary delays like this

Second that. It's only because I can consistently reproduce it that I proposed it.

While at it, I noticed that `mEmailInput` changed from `private` to `protected`. Any particular reason behind the change? If none, let's keep it as `private`.

Reply:
Using `mEmailInput` in the delayed runnable causes the following lint warning.
> Access to private member of class 'LoginEmailFragment' requires synthetic accessor call less... (⌘F1) 
> 
> This inspection is intended for J2ME and other highly resource constrained environments. Applying the results of this inspection without consideration might have negative effects on code clarity and design.
> 
> Reports references to non-constant private members of a different class, for which javac will generate a package-private synthetic accessor method.
> 
> An inner class and its containing class are compiled to separate class files. The Java virtual machine normally prohibits access from a class to private fields and methods of another class. To enable access from an inner class to private members of a containing class or the other way around javac creates a package-private synthetic accessor method. Less use of memory and greater performance may be achieved by making the member package-private, thus allowing direct access without the creation of a synthetic accessor method.

Changing `mEmailInput` to `protected` resolves that.